apps/concierge_site/assets/mjml/notification.mjml

```diff
@@ -99,10 +99,10 @@
     <mj-raw>
       <!-- Footer -->
     </mj-raw>
-    <mj-section padding="0">
+    <mj-section padding="0" css-class="footer">
```

Comment:
I'm not familiar with a `css-class` attribute, what does it do?

Reply:
It adds that class to the outermost element. This is actually a remnant of the mj-html-attributes approach I tried first, which relies on having a selector. I could remove this if we stick with adding the `role` to the `<p>` tag inside this section.

Reply:
Oh, cool! Hadn't seen it before.